avocado/utils/network/interfaces.py

```diff
@@ -595,3 +595,14 @@ def are_packets_lost(self, peer_ip, options=None, sudo=False):
         except Exception as ex:
             msg = f"Failed to ping. {ex}"
             raise NWException(msg)
+
+    def netmask_to_cidr(self, netmask):
+        """ Function is used to check the netmask value and convert
```

Comment:
insert a empty line after, the reason why travis is failing.. refer to other methods doc string inthis file